tests/unit/src/test_conditionsStore.cpp

```diff
@@ -415,7 +391,7 @@ SCENARIO( "Providing derived conditions", "[ConditionStore][DerivedConditions]"
 			THEN( "readonly providers should not perform erase actions" )
 			{
 				mockProvNamed.SetRONamedProvider(store, "named1");
-				REQUIRE_FALSE( store.Erase("named1") );
+				REQUIRE_FALSE( store.Set("named1", 0) );
```

Comment:
This should probably be changed to testing that read-only providers reject "Set"ting their values.

Reply:
Agreed, I updated the title of the test.